generator-tools/templates/reducer/REDUCER_NAME.actions.js.tpl

```diff
@@ -0,0 +1,11 @@
+import { createAction } from 'redux-act';
+
+export const ensure<%= componentNameCamelCase %> = createAction('Retrieve <%= componentNameCamelCase %>');
+export const <%= componentNameCamelCase %>Retrieved = createAction('<%= componentNameCamelCase %> successfully retrieved.');
```

Comment:
This currently produces the first letter capitalized. Maybe we can add the following case:
`componentNameCamelCaseDecapitalized: S.camelize(componentName)`